### PR TITLE
Add group policy package assignment documentation

### PR DESCRIPTION
**Pull request by icchan8**
Feature will be going into TAP in the next two weeks

### CODE REVIEW

File: teams/teams-ps/teams/Grant-CsGroupPolicyPackageAssignment.md
Comment (dariomws) [inline teams/teams-ps/teams/Grant-CsGroupPolicyPackageAssignment.md:82]:
```suggestion
The policy rankings for each of the policy types in the package. To specify the policy rankings, follow this format: "\<PolicyType\>, \<PolicyRank\>". Delimiters of ' ', '.', ':', '\t' are also acceptable. Supported policy types of a policy package are listed [here](https://docs.microsoft.com/MicrosoftTeams/manage-policy-packages#what-is-a-policy-package). Policy rank should be a number greater than or equal to 1.
```

**Review by dariomws — APPROVED** [review]
Thanks @icchan8!

I found only a minor detail.

**Comment by dariomws** [comment]
@icchan8 Will this cmdlet be available in 1.1.7-preview?

File: teams/teams-ps/teams/Grant-CsGroupPolicyPackageAssignment.md
Comment (get-itips) [inline teams/teams-ps/teams/Grant-CsGroupPolicyPackageAssignment.md:28]:
```suggestion
This cmdlet assigns a policy package to a group in a tenant. The available policy packages and their definitions can be found by running Get-CsPolicyPackage. Policy rankings can be optionally specified to determine which policies in a package will be assigned to the user in case they belong to two or more groups. If policy rankings for a policy type is not specified, the group's policy assignment for that policy type will be ranked last. Finally, if a user was directly assigned a package, direct assignment takes precedence over group assignment. For more information on policy rankings and group policy assignments, please review [the description section under New-CsGroupPolicyAssignment](New-CsGroupPolicyAssignment.md#description). For more information on policy packages, please review [Manage policy packages in Microsoft Teams](https://docs.microsoft.com/MicrosoftTeams/manage-policy-packages).
```

**Comment by tseward** [comment]
Approved pending above changes.

**Comment by icchan8** [comment]
> @icchan8 Will this cmdlet be available in 1.1.7-preview? Or only TAP?

It will be available in the preview but only TAP tenants can use it

**Review by get-itips — APPROVED** [review]

**Comment by icchan8** [comment]
Please don't merge yet. There may be a few more clarifications that need to be added.

**Comment by icchan8** [comment]
@tseward Documentation is ready. Could you help with approving and merging?

**Review by tseward — APPROVED** [review]

**Comment by icchan8** [comment]
@Kateyanne Please help with merging, thanks

**Pull request merged**
